src/sentry/static/sentry/app/views/performance/transactionSummary/content.tsx

```diff
@@ -204,22 +202,27 @@ class SummaryContent extends React.Component<Props, State> {
               eventView={eventView}
               totalValues={totalCount}
             />
-            <TransactionsList
-              location={location}
-              organization={organization}
-              eventView={eventView}
-              selected={selectedSort}
-              options={sortOptions}
-              titles={[t('id'), t('user'), t('duration'), t('timestamp')]}
-              handleDropdownChange={this.handleTransactionsListSortChange}
-              generateLink={{
-                id: generateTransactionLink(transactionName),
-              }}
-              baseline={transactionName}
-              handleBaselineClick={this.handleViewDetailsClick}
-              handleCellAction={this.handleCellAction}
-              handleOpenInDiscoverClick={this.handleDiscoverViewClick}
-            />
+            {totalValues === null ? (
+              <Placeholder height="352px" />
+            ) : (
+              <TransactionsList
+                location={location}
+                organization={organization}
+                eventView={eventView}
+                titles={[t('id'), t('user'), t('duration'), t('timestamp')]}
+                handleDropdownChange={this.handleTransactionsListSortChange}
+                generateLink={{
+                  id: generateTransactionLink(transactionName),
+                }}
+                baseline={transactionName}
+                handleBaselineClick={this.handleViewDetailsClick}
+                handleCellAction={this.handleCellAction}
+                handleOpenInDiscoverClick={this.handleDiscoverViewClick}
+                {...getTransactionsListSort(location, {
```

Comment:
@alxu-sentry It'll be better to relocate `getTransactionsListSort()` function into the same file as the `<TransactionsList/>` component.

Reply:
Are you sure? Only Releases and Transaction Summary use TransactionsList at the moment, in pretty different ways. I think it'd be better to keep it here until we have more users and a clearer view of what a unified API would look like.